_sections/live-share-code-with-me-support-vscode.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+#### Starting a Live Share Session 
+If you have the Live Share extension installed on VS Code, you have the option of using Live Share to pair program on your call. To start a Live Share session, you should select the Live Share tab on the Share Code popup modal, and select the project you would like to share. 
```

Comment:
doesnt depend on installation i think but only on the fact that host wants to share a vs code project, but assert this please